okio/src/jvmTest/kotlin/okio/internal/ResourceFileSystemTest.kt

```diff
@@ -48,6 +50,7 @@ class ResourceFileSystemTest {
     assertThat(metadata.isDirectory).isFalse()
 
     val content = fileSystem.read(path) { readUtf8() }
+    assertThat(fileSystem.metadata(path).isRegularFile).isTrue()
```

Comment:
I added some extra tests for metadata() because the read tests no longer exercise as much Okio code.